.github/workflows/unit-tests.yaml

```diff
@@ -0,0 +1,49 @@
+name: 'Unit Tests'
```

Comment:
This can probably be added to continuous-integrations workflow instead of creating another workflow.